contrib/ci/fedora.sh

```diff
@@ -5,6 +5,10 @@ set -x
 #get any missing deps from the container
 ./contrib/ci/fwupd_setup_helpers.py install-dependencies --yes -o fedora
 
+dnf install -y \
+    https://people.freedesktop.org/~hughsient/temp/libgusb-0.4.5-0.218.20230203git.fc37.x86_64.rpm \
+    https://people.freedesktop.org/~hughsient/temp/libgusb-devel-0.4.5-0.218.20230203git.fc37.x86_64.rpm
+
```

Comment:
still want to block merging until these are in

Reply:
Ack, agree.

Reply:
Done, updates are in normal Fedora now.